src/dpdb/dpdb.js

```diff
@@ -19,7 +19,8 @@ var DPDB_CACHE = require('./dpdb-cache.js');
 var Util = require('../util.js');
 
 // Online DPDB URL.
-var ONLINE_DPDB_URL = 'https://storage.googleapis.com/cardboard-dpdb/dpdb.json';
+var ONLINE_DPDB_URL =
+  'https://webvrrocks.github.io/webvr-polyfill-dpdb/dpdb.json';
```

Comment:
I just created a new subdomain for this, and apologies for asking for yet another change (!), could you change this URL to `https://dpdb.webvr.rocks/dpdb.json`?